src/utils/contextUtils.ts

```diff
@@ -83,3 +83,18 @@ export function getPathToWorkspaceStorage(...args: string[]): string | undefined
     fse.ensureDirSync(EXTENSION_CONTEXT.storagePath);
     return path.join(EXTENSION_CONTEXT.storagePath, ...args);
 }
+
+export async function trustWrapper(mvnw: string): Promise<boolean> {
+    const key: string = "trustMavenWrapper";
+    const trust: boolean | undefined = EXTENSION_CONTEXT.workspaceState.get<boolean | undefined>(key);
+    if (trust === undefined) {
+        const msg: string = `Maven Wrapper is found in this workspace, do you trust it? (${mvnw})`;
+        const YES: string = "Yes";
+        const NO: string = "No";
+        const choice: string | undefined = await window.showInformationMessage(msg, YES, NO);
+        EXTENSION_CONTEXT.workspaceState.update(key, choice === YES);

```

Comment:
What happened when i click `esc`?

Reply:
same as `No`.  it fallbacks to search for `mvn` installed locally.

Reply:
Do you want me to keep poping it up on next time, if the user press `esc` ?